### PR TITLE
task/DES-2758, DES-2747: Tapis v3 Job Notifications

### PR DESCRIPTION
**Pull request by rstijerina**
## Overview: ##

- Display toast notifications for v3 job status updates
- Disable v2 angular toast notifications

## PR Status: ##

* [X] Ready.

## Related Jira tickets: ##

* [DES-2758](https://tacc-main.atlassian.net/browse/DES-2758)
* [DES-2747](https://tacc-main.atlassian.net/browse/DES-2747)

## Testing Steps: ##
1. Submit a job
2. Confirm toasts show in bottom left corner

## UI Photos:
Screenshot 2024-05-20 at 2 19 29 PM
Screenshot 2024-05-20 at 2 19 19 PM

## Notes: ##
TODO items for future:
- [x] Fix bug where on job status change, focus redirects to where the notification context is instantiated
- [ ] Style notification toast popups
- [ ] Hook job status updates via websocket events into updating the job listing and job detail modal

### CODE REVIEW

**Review by chandra-tacc — APPROVED** [review]
LGTM. one comment on some dead code. 
While my ngrok was running, I saw notifications coming through in ngrok logs.

File: designsafe/static/scripts/ng-designsafe/providers/notifications-provider.js [hunk 1]
```diff
@@ -148,16 +148,16 @@ function NotificationService(
         });
 
         const toastViewLink = renderLink(msg);
-        if (typeof toastViewLink !== 'undefined') {
```
Comment (chandra-tacc) [inline designsafe/static/scripts/ng-designsafe/providers/notifications-provider.js:151]: Is this code relevant to v3 ?

Comment (rstijerina) [reply designsafe/static/scripts/ng-designsafe/providers/notifications-provider.js:151]: It's actually relevant to the old code, but this code is still "live" on the v3 page and was causing duplicate toast notifications from this old code. We'll need to rip this all out in time, but short term this disables the old code from producing notifications
